requirements.txt

```diff
@@ -1,6 +1,7 @@
 click >= 6.6
 cloudpickle >= 1.5.0
 dask == 2021.07.2
+jinja2
```

Comment:
You should add this to the `continuous_integration/environment_py3*.py` files as well. Jinja should already be included as a transitive dependency but the conda solver may create different results if it is a first level dependency.

Reply:
Thanks @fjetter. We discussed adding `jinja2` as a top level dependency at the maintainers standup a few months ago and the consensus was to go ahead. I missed the CI environment though.

Reply:
We also have HTML reprs in `dask` which would benefit from this, so we may need to make `jinja2` a dependency of `dask` too.

Reply:
I probably wasn't, yet, around during that developer meeting but I don't have any objections to having it as a dependency. I think jinja2 has been around for long enough to consider it stable and it's not very invasive. all good.

The templates are much nicer to read! Also, we could even enable linters and/or formatters for this now :) one step at a time, though.

Reply:
> Also, we could even enable linters and/or formatters for this now :) one step at a time, though.

Yeah, just having my editor do hightlighting properly is awesome. GitHub too.

We could totally look at introducing prettier though.